src/main/resources/META-INF/resources/primefaces/datepicker/0-datepicker.js

```diff
@@ -110,6 +112,22 @@
                 $.extend(this.options.locale, this.options.userLocale);
             }
 
+            if(this.options.showWeek && !this.options.weekCalculator) {
```

Comment:
I think you can remove this if statement. Users can calculate week numbers using a custom weekCalculator function;
```
function myWeekCalculator(date) {
    ...
    return weekNo;
}

<p:datePicker weekCalculator="myWeekCalculator"
```

Reply:
No I can't remove it. This if checks if a custom week calculator is provided or if some internal defaults have to be configured. But only if `showWeek` is enabled. If the user provides a custom week calculator I don't have to check if the internals are correctly configured because it is totally up to the user.